### ISSUE TITLE
Broken link in run_prettify.min.js from CloudFlare

### ISSUE THREAD

**Issue by KhronosWebservices** [KhronosGroup/OpenXR-Registry#1]
The OpenXR specification Version 1.0.19 suffers the same issue as the KTX Specification as outlined in this issue https://github.com/KhronosGroup/KTX-Specification/issues/201. 

The included JS file run_prettify.min.js from CloudFlare is pointing to Google Code (https://google-code-prettify.googlecode.com/svn/loader/prettify.css). It appears that this JS library is archived and no longer maintained. If possible, this repository should move to a different JS library to handle the code prettifying.

**Comment by rpavlik** [comment]
yeah, I was waiting to see how the migration to something else shook out for Vulkan. Nothing quite dropped-in the same way.
